docs/meta/correlations.md

```diff
@@ -0,0 +1,300 @@
+- `type`: The type of correlation to be used ([see types below](#types-of-correlations)).
+- `rules`: A list of Sigma rules that are used for the correlation (either by name or ID).
+- `group-by`: _(Optional)_ A list of fields to group the events by.
+- `timespan`: The time frame in which the events are aggregated.
+- `condition`: The condition that has to be met for the correlation to match.
+
+Below is an example of a correlation rule that detects multiple failed logons for a single user within a 5-minute time frame.
+
+```yaml
+title: Multiple failed logons for a single user (possible brute force attack)
+status: test
+correlation: // [!code ++] // [!code focus:10]
+    type: event_count
+    rules:
+        - failed_logon
+    group-by:
+        - TargetUserName
+        - TargetDomainName
+    timespan: 5m
+    condition:
+        gte: 10
+tags:
+    - brute_force
+    - attack.t1110
+```
+
+Because this correlation rule references another Sigma rule called `failed_logon`, a rule with the field `name: failed_logon` needs to be supplied alongside this rule when we're converting the correlation rule for our SIEM.
+
+Therefore, it's common to place this "base" Sigma rule in the same file as the correlation rule, using the `---` separator to separate the two rules.
+
+::: code-group
+
+```yaml:line-numbers [rules/windows_failed_login_single_user.yml]
+title: Windows Failed Logon Event
+name: failed_logon # Rule Reference // [!code ++]
+description: Detects failed logon events on Windows systems.
+logsource:
+    product: windows
+    service: security
+detection:
+    selection:
+        EventID: 4625
+    condition: selection
+---
+title: Multiple failed logons for a single user (possible brute force attack)
+correlation:
+    type: event_count
+    rules:
+        - failed_logon # Referenced here  // [!code highlight]
+    group-by:
+        - TargetUserName
+        - TargetDomainName
+    timespan: 5m
+    condition:
+        gte: 10
+```
+
+:::
+
+Converting the above "Multiple failed logons for a single user" Sigma rule to Splunk SPL would yield the following query:
+
+```bash
+sigma convert -t splunk -p splunk_windows \
+    rules/windows_failed_login_single_user.yml
+```
+
+```splunk
+source="WinEventLog:Security" EventCode=4625
+| bin _time span=5m
+| stats count as event_count by _time TargetUserName TargetDomainName
+| search event_count >= 10
+```
+
+::: warning Required Reference
+
+Whilst [Sigma Filters](/docs/meta/filters) meta rules won't require you to supply the referenced Sigma rule, <u>**Sigma Correlations will**</u> enforce that the referenced Sigma rule is present either in the same Sigma file, or supplied as a separate Sigma rule file when converting.
+
+```text
+Error: Error while conversion: Rule 'failed_logon' not found in rule collection
+```
+
+It's recommended to keep the referenced Sigma rule in the same file as the correlation rule to ensure that the correlation rule can be easily shared and understood by others.
```

Comment:
I am more to somethink like : "If the rule is only for this correlation , it 's recommended"
I don't want to give the impression that it's an "obligation"